### PR TITLE
Update dependencies and tests for testing middleware fix

### PR DESCRIPTION
**Pull request by KevinGrandon**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ onerror = function(m,s,l,c,e) {
 };
 </script>`;
           ctx.template.head.unshift(script);
-        } else if (ctx.path === ctx.prefix + '/_errors') {
+        } else if (ctx.path === '/_errors') {
```
Comment (dennisgl) [inline src/server.js:62]: Don't we want the prefix?

Comment (KevinGrandon) [reply src/server.js:62]: I was told that we strip the prefix, but it looks like we might only strip it for the URL, and not path? 

https://github.com/fusionjs/fusion-cli/blob/master/lib/strip-prefix.js

Comment (KevinGrandon) [reply src/server.js:62]: (Path is probably derived from the URL, but will do some verification)

Comment (dennisgl) [reply src/server.js:62]: Cool, almost forget about the stripping

**Review by dennisgl — APPROVED** [review]

**Comment by KevinGrandon** [comment]
!merge

**Pull request merged**